### ISSUE TITLE
CogVideoX-5b-I2V模型报错（刚入门菜鸟求老师答疑）

### ISSUE THREAD

**Issue by wx331406** [THUDM/CogVideo#543]
### System Info / 系統信息

win10

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

import torch
from diffusers import CogVideoXImageToVideoPipeline
from diffusers.utils import load_image
from PIL import Image
import random
import gc
import gradio as gr
import uuid

# 配置设备
device = "cuda" if torch.cuda.is_available() else "cpu"

# 全局加载模型以提高效率
print("Loading model...")
global_pipe = CogVideoXImageToVideoPipeline.from_pretrained(
    "D:\\DreamForge AI\\models\\CogVideoX-5b-I2V",
    torch_dtype=torch.float16,
).to(device)

def generate_video_from_image(
    prompt: str,
    image: Image.Image,
    num_inference_steps: int = 50,
    guidance_scale: float = 7.5,
    seed: int = -1,
):
    if seed == -1:
        seed = random.randint(0, 2**31 - 1)

    # 验证输入图像
    if image is None or not isinstance(image, Image.Image):
        raise ValueError("Invalid image input. Please upload a valid image.")

    # 调整图像尺寸
    image = image.resize((720, 480))  # 确保尺寸与模型一致
    image = load_image(image)

    # 设置随机生成器
    generator = torch.manual_seed(seed)

    # 开始生成视频
    result = global_pipe(
        image=image,
        prompt=prompt,
        num_inference_steps=num_inference_steps,
        num_videos_per_prompt=1,
        guidance_scale=guidance_scale,
        generator=generator,
        output_type="mp4",
    )

    # 动态生成唯一文件名
    output_path = f"generated_video_{uuid.uuid4().hex[:8]}.mp4"
    with open(output_path, "wb") as f:
        f.write(result.videos[0])

    return output_path, seed

# Gradio 接口
def gradio_interface(prompt, image, steps, scale, seed):
    """封装 Gradio 接口的调用逻辑"""
    video_path, seed_used = generate_video_from_image(
        prompt=prompt,
        image=image,
        num_inference_steps=steps,
        guidance_scale=scale,
        seed=seed,
    )
    return video_path, f"Seed used: {seed_used}"

# 设置 Gradio 界面
with gr.Blocks() as demo:
    gr.Markdown("# 图生视频生成器 (CogVideoX-5b-I2V)")
    gr.Markdown("通过上传图像并输入提示词生成短视频。")

    with gr.Row():
        with gr.Column():
            prompt = gr.Textbox(label="提示词 (Prompt)", value="A sunset over the ocean with waves gently lapping")
            image = gr.Image(type="pil", label="输入图像")
            steps = gr.Slider(10, 100, step=10, value=50, label="推理步数 (Inference Steps)")
            scale = gr.Slider(1.0, 10.0, step=0.5, value=7.5, label="引导比例 (Guidance Scale)")
            seed = gr.Number(value=-1, label="随机种子 (Seed, -1 表示随机)")

        with gr.Column():
            output_video = gr.Video(label="生成的视频 (Generated Video)")
            output_seed = gr.Textbox(label="生成使用的种子 (Seed)", interactive=False)

    generate_button = gr.Button("生成视频 (Generate Video)")

    # 绑定事件
    generate_button.click(
        gradio_interface,
        inputs=[prompt, image, steps, scale, seed],
        outputs=[output_video, output_seed],
    )

# 启动 Gradio 应用
if __name__ == "__main__":
    demo.launch(share=True)

### Expected behavior / 期待表现

PS D:\DreamForge AI>  & 'd:\DreamForge AI\nenv\Scripts\python.exe' 'c:\Users\Administrator\.vscode\extensions\ms-python.debugpy-2024.12.0-win32-x64\bundled\libs\debugpy\adapter/../..\debugpy\launcher' '60326' '--' 'D:\DreamForge AI\cs.py' 
Loading model...
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:02<00:00,  1.02s/it]
Loading pipeline components...: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:05<00:00,  1.14s/it]
信息: 用提供的模式无法找到文件。
* Running on local URL:  http://127.0.0.1:7860

**Comment by zRzRzRzRzRzRzR** [comment]
什么意思，具体报错事
